backend/api/pretix/mutations.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import strawberry
+from strawberry.types import Info
+
+import pretix
+from api.permissions import IsAuthenticated
+from api.pretix.query import get_user_tickets
+from api.pretix.types import AttendeeTicket, UpdateAttendeeTicketInput
+from conferences.models.conference import Conference
+
+logger = logging.getLogger(__name__)
+
+
+@strawberry.type
+class UpdateAttendeeTicketError:
+    message: str = ""
+
+
+@strawberry.type
+class TicketReassigned:
+    id: strawberry.ID
+    message: str = "Ticket was Successfully reassigned to {email}"
+
+
+UpdateAttendeeTicketResult = strawberry.union(
+    "UpdateAttendeeTicketResult",
+    (TicketReassigned, AttendeeTicket, UpdateAttendeeTicketError),
+)
+
+
+@strawberry.type
+class AttendeeTicketMutation:
+    @strawberry.mutation(permission_classes=[IsAuthenticated])
+    def update_attendee_ticket(
+        self, info: Info, conference: str, input: UpdateAttendeeTicketInput
+    ) -> UpdateAttendeeTicketResult:
+        conference = Conference.objects.get(code=conference)
+
+        try:
+            pretix.update_ticket(conference, input)
+
+            # TODO: filter by orderposition
+            tickets = get_user_tickets(
+                conference, info.context.request.user.email, language="en"
+            )
+
+            tickets = list(filter(lambda ticket: str(ticket.id) == input.id, tickets))
+            if tickets:
+                return tickets[0]
+
```

Comment:
I don't understand why we need this, doesn't Pretix return you the updated order position in the original update_ticket call?

Reply:
Yes, but not will all the information we need: the item it's just an ID instead of the dictionary, the question as well... 
in order to return AttendeeTicket type I need that information... 
At least the questions...